pkg/sync/http/http_sync.go

```diff
@@ -15,15 +15,13 @@ import (
 )
 
 type Sync struct {
-	URI          string
-	Client       Client
-	Cron         Cron
-	BearerToken  string
-	LastBodySHA  string
-	Logger       *logger.Logger
-	ProviderArgs sync.ProviderArgs
-
-	ready bool
+	URI         string
+	Client      Client
+	Cron        Cron
+	LastBodySHA string
+	Logger      *logger.Logger
+	ready       bool
```

Comment:
Sorry for nitpicking, shall we separate **P**ublic and **p**rivate fields 

```suggestion
	URI         string
	Client      Client
	Cron        Cron
	LastBodySHA string
	Logger      *logger.Logger
	BearerToken string
	
	ready       bool
```